melodic/distribution.yaml

```diff
@@ -1860,6 +1860,24 @@ repositories:
       url: https://github.com/ros/common_tutorials.git
       version: indigo-devel
     status: maintained
+  compass:
```

Comment:
This name seems pretty generic to me, and thus I think it probably violates https://www.ros.org/reps/rep-0144.html#global-guidelines point 1 or 4. It's kind of subjective, as this is a generic package, but it isn't clearly used by large parts of the community. Thus I think this package would be a candidate for prefixing the name of the entity.

Reply:
What about keeping compass_msgs and renaming compass to magnetometer_compass? 

There's really no need for org prefix as the package just implements a generic algorithm anybody would use when trying to get azimuth readings from a magnetometer.

Reply:
@clalancette I guess this has somehow fallen off the table? There doesn't seem to be much time until the sync...